osler/assets/util/CSRFToken.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import getCookie from './getCookie';
+
+const csrftoken = getCookie('csrftoken');
+
+const CSRFToken = () => {
+    return (
+        <input type="hidden" name="csrfmiddlewaretoken" value={csrftoken} />
+    );
+};
+export default CSRFToken;
```

Comment:
Find some way to extract CSRF token that does not involve cookies since they will soon be disabled in Chrome (this may or may not be possible)